### PR TITLE
Serialize Arrays

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: packages/as-types/assembly/serialization/arrays.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// @ts-ignore: decorator
+@inline @lazy const MIN_SIZE: usize = 8;
```
Comment (gregLibert) [inline packages/as-types/assembly/serialization/arrays.ts:2]: Why 8 ?

**Review by gregLibert — APPROVED** [review]
:question:

File: packages/as-types/assembly/serialization/arrays.ts (hunk 1 above)
Comment (Thykof) [reply packages/as-types/assembly/serialization/arrays.ts:2]: I copied this from assemblyscript array

Comment (peterjah) [reply packages/as-types/assembly/serialization/arrays.ts:2]: weird :/

Comment (peterjah) [reply packages/as-types/assembly/serialization/arrays.ts:2]: this seems to work just fine (with your unit tests)
```
export function arrayToBytes<T>(array: T[]): StaticArray<u8> {
  return changetype<StaticArray<u8>>(array);
}

export function bytesToArray<T>(array: StaticArray<u8>): T[] {
  return changetype<T[]>(array);
}

**Review by peterjah — CHANGES_REQUESTED** [review]

File: packages/as-types/assembly/serialization/arrays.ts (hunk 1 above)
Comment (gregLibert) [reply packages/as-types/assembly/serialization/arrays.ts:2]: Yes, but the problem here is that both have the same pointer...

**Review by gregLibert — APPROVED** [review]
:clap:

**Review by peterjah — APPROVED** [review]

**Pull request merged**